js/api/api.js

```diff
@@ -0,0 +1,20 @@
+const API = (() => {
+  const URL = 'https://bootcamp-api.codeit.kr/';
+  const URL_API = 'api';
+  const CHECK_EMAIL = '/check-email';
+  const SIGN_IN = '/sign-in';
+  const SIGN_UP = '/sign-up';
+  return {
+    getCheckEmailURl() {
+      return URL + URL_API + CHECK_EMAIL;
+    },
+    getSignInUrl() {
+      return URL + URL_API + SIGN_IN;
+    },
+    getSignUpUrl() {
+      return URL + URL_API + SIGN_UP;
+    },
+  };
+})();
+
+export default API;
```

Comment:
```javascript
const URL = 'https://bootcamp-api.codeit.kr/';
const URL_API = 'api';
const CHECK_EMAIL = '/check-email';
const SIGN_IN = '/sign-in';
const SIGN_UP = '/sign-up';

const getCheckEmailURl = () => {
  return URL + URL_API + CHECK_EMAIL;
};
const getSignInUrl = () => {
  return URL + URL_API + CHECK_EMAIL;
};
const getSignUpUrl = () => {
  return URL + URL_API + CHECK_EMAIL;
};

export { getCheckEmailURl, getSignInUrl, getSignUpUrl }
```
기존에 사용하던 위 방식과 고민을 많이 하다가 즉시실행함수를 한번 사용해서 구현해 봤습니다.
어떤 방식으로 export하면 좋을 지 모르겠습니다.

현재 push된 버전은 안 쓰게 되는 함수도 같이 return 돼서 안 좋지만 한 곳에서 import가 많이 필요하면 라인이 계속 늘어나서 보기 안 좋다는 느낌을 받았습니다.

Reply:
넵넵. 지금 상황에서는 매개변수나 별도의 로직이 필요 없으므로 함수는 필요 없을 것으로 보여요.
다음과 같은 코드는 어떨까요?:
```suggestion
const URL = 'https://bootcamp-api.codeit.kr/';
const URL_API = 'api';
const CHECK_EMAIL = '/check-email';
const SIGN_IN = '/sign-in';
const SIGN_UP = '/sign-up';

export {
  CHECK_EMAIL_URL: URL + URL_API + CHECK_EMAIL,
  SIGNIN_URL: URL_API + SIGN_IN,
  SIGNUP_URL: URL_API + SIGN_UP
}
```